elements/pfe-avatar/src/pfe-avatar.js

```diff
@@ -7,6 +7,48 @@ class PfeAvatar extends PFElement {
     return "pfe-avatar";
   }
 
+  static get properties() {
+    return {
+      name: {
+        title: "Username",
+        type: String,
+        observer: "_updateWhenConnected",
+        default: "",
```

Comment:
If the default is empty, does the property need to be set?

Reply:
In general, no, but avatar wants there to always be a name, so it defaults to "".